core_helpers/mobile_app_helper.py

```diff
@@ -135,7 +135,7 @@ def swipe_to_element(self,scroll_group_locator, search_element_locator, max_swip
                         result_flag = True
                         return result_flag
                 except Exception:
-                    self.write('Element not found, swiping again', 'debug')
+                    self.write('Element not found, swiping again', 'critical')
                     pass
```

Comment:
Can you please create an issue to remove the `pass` statements after the `self.write` step in the `except` block in `mobile_app_helper` module & other modules?

Reply:
Opened issue https://github.com/qxf2/qxf2-page-object-model/issues/478